app/models/series.rb

```diff
@@ -536,8 +536,9 @@ def Series.load_from_download(handle, options, cached_files = nil)
   def Series.load_from_file(file, options, cached_files = nil)
     file.gsub! ENV['DEFAULT_DATA_PATH'], ENV['DATA_PATH']
     %x(chmod 766 #{file}) unless file.include? '%'
-    dp = DownloadProcessor.new('manual', options.merge(:path => file))
+    dp = DownloadProcessor.new('manual', options.merge!(:path => file))
     series_data = dp.get_data
+    options.delete(:path) ## don't include :path in the description. But don't remove the bang above! It is needed.
```

Comment:
This makes me think the DownloadProcessor constructor needs a separate path argument.

Reply:
Let me look into that. Might cause more trouble than the "trouble" of line 541 above.